lib/Distribution/Opensuse/Tumbleweed.pm

```diff
@@ -163,4 +164,8 @@ sub get_encrypted_volume_activation {
     return Installation::SystemProbing::EncryptedVolumeActivationController->new();
 }
 
+sub get_install_back_popup_controller {
```

Comment:
we have some lack of consistency with other method (we will fix that later), for now let's not add '_controller' in the name of the method (it doesn't make sense because they all are controllers after all :) )